client/my-sites/plans-features-main/index.jsx

```diff
@@ -551,13 +549,38 @@ export class PlansFeaturesMain extends Component {
 						plans={ visiblePlans }
 					/>
 				) }
-				{ shouldShowPlansFeatureComparison ? this.showFeatureComparison() : this.getPlanFeatures() }
+				{ this.hideEcommerceWrapper() }
 				{ this.renderProductsSelector() }
 				{ this.mayRenderFAQ() }
 			</div>
 		);
 	}
 
+	hideEcommerceWrapper() {
+		const { isLaunchPage, shouldShowPlansFeatureComparison } = this.props;
+		if ( ! isLaunchPage ) {
+			return shouldShowPlansFeatureComparison
+				? this.showFeatureComparison()
+				: this.getPlanFeatures();
+		}
+
+		return (
+			<ProvideExperimentData name="hide_ecommerce_launch_site">
```

Comment:
I recommend adding the new `isEligible` prop to check if it's the launch page - check this [review comment](https://github.com/Automattic/wp-calypso/pull/50512/files#r615207493) on a different PR on how it works.